lldb/bindings/interface/SBValueDocstrings.i

```diff
@@ -45,6 +45,36 @@ SBValue instance which interprets the value object as representing the head of a
 linked list."
 ) lldb::SBValue;
 
+%feature("docstring", "
+Returns true if the SBValue holds any useful state
+and false otherwise.
+IsValid is a very weak API, lldb will only return
```

Comment:
Suggestion: "weak" -> "limited"
In my opinion, weak sounds like a value judgement as opposed to limited which sounds more neutral.

Reply:
I was thinking of weak vrs. strong assertions, but limited is also fine.